src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/Native_x64_VSCode_Rust_DebugInfo_2.exe.sarif

```diff
@@ -55,7 +55,28 @@
+              "clang LLVM (rustc version 1.55.0 (c8dfcfe04 2021-09-06)) : masm : 12001.0.0.0 : liballoc-e1d125b3a421e009.rlib [SHA1] (alloc-e1d125b3a421e009.alloc.b6d975c0-cgu.0.rcgu.o)\r\nclang LLVM (rustc version 1.55.0 (c8dfcfe04 2021-09-06)) : masm : 12001.0.0.0 : libcompiler_builtins-55394bc732b75eeb.rlib [SHA1] (compiler_builtins-55394bc732b75eeb.compiler_builtins.4be54af4-cgu.116.rcgu.o,compiler_builtins-55394bc732b75eeb.compiler_builtins.4be54af4-cgu.30.rcgu.o,compiler_builtins-55394bc732b75eeb.compiler_builtins.4be54af4-cgu.93.rcgu.o)\r\nclang LLVM (rustc version 1.55.0 (c8dfcfe04 2021-09-06)) : masm : 12001.0.0.0 : libcore-f309332ab230c371.rlib [SHA1] (core-f309332ab230c371.core.8174c79a-cgu.0.rcgu.o)\r\nclang LLVM (rustc version 1.55.0 (c8dfcfe04 2021-09-06)) : masm : 12001.0.0.0 : libpanic_unwind-c3aa48ffdde66347.rlib [SHA1] (panic_unwind-c3aa48ffdde66347.panic_unwind.932dc30b-cgu.0.rcgu.o)\r\nclang LLVM (rustc version 1.55.0 (c8dfcfe04 2021-09-06)) : masm : 12001.0.0.0 : librustc_demangle-cd2025f252bca17e.rlib [SHA1] (rustc_demangle-cd2025f252bca17e.rustc_demangle.7bc06e87-cgu.0.rcgu.o)\r\nclang LLVM (rustc version 1.55.0 (c8dfcfe04 2021-09-06)) : masm : 12001.0.0.0 : libstd-ac1aca38635ef0b8.rlib [SHA1] (std-ac1aca38635ef0b8.std.b81c7b07-cgu.0.rcgu.o)\r\n\r\n"
+            ]
+          },
+          "locations": [
+            {
+              "physicalLocation": {
+                "artifactLocation": {
+                  "uri": "file:///Z:/src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Native_x64_VSCode_Rust_DebugInfo_2.exe",
+                  "index": 0
+                }
+              }
+            }
+          ]
+        },
+        {
+          "ruleId": "BA2004",
```

Comment:
>ruleId [](http://example.com/codeflow?start=11&length=6)

this looks like a bug, I can't see why this new result has popped up, will debug.

Reply:
saw this file was checked in by me so I get the branch and debugged,
found the small issue and fixed and pushed, please see my last commit if correct.

Reply:
YES. Thanks for fixing this! I really appreciate it.